store/src/pmmr.rs

```diff
@@ -138,19 +138,27 @@ where
 		if self.rm_log.includes(position) {
 			return None;
 		}
+
 		// ... or in the prune list
-		let prune_shift = match self.pruned_nodes.get_leaf_shift(position) {
-			Some(shift) => shift,
-			None => return None,
-		};
+		// let prune_shift = match self.pruned_nodes.get_leaf_shift(position) {
```

Comment:
Data file pruning/compaction commented out for now.
Tackle this in a separate PR once we are comfortable that hash file compaction is working.